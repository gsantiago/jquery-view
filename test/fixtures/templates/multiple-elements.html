<div>
  <span class="name">{{ firstname }}</span>
  <p>
    {% description %}
  </p>
  <a href="mailto:{{user.email}}">{{ user.name }}</a>
</div>