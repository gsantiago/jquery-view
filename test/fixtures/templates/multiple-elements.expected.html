<div>
  <span class="name">Guilherme</span>
  <p>
    <strong>Front-end Developer</strong>
  </p>
  <a href="mailto:guilherme@email.com">Guilherme Santiago</a>
</div>